docs/dcp2_system_design.rst

```diff
@@ -1223,49 +1247,117 @@ When a data release first enters the preparation phase, it is identical to the
 data release that precedes it. In other words, it is made up of the same set of
 snapshots as the previous data release. After that, and until the release is
 published, the (meta)data in the data release being prepared is subject to
-*alterations*. Alterations to (meta)data should generally be made in the form of
-delta staging areas. A delta staging area has the ``is_delta`` property set to
-``true``. The layout of delta staging areas and the rules for importing them
-differ slightly from those for non-delta staging areas i.e. those with a
-``is_delta`` value of ``false``. Only delta staging areas may indicate the
-removal of (meta)data or the deletion of data, non-delta staging areas must not.
+*alterations*.
+
+
+Coordination between source and importer
+-----------------------------------------
+
+A staging area may only be modified in between importer invocations, not while
+the importer is running. Coordination of access to a staging area occurs out of
+band e.g. via Slack or a ticketing system.
+
+
+Alterations via ``complete`` staging areas
+------------------------------------------
+
+|nn| Before the ``delta`` staging areas specification was written, the DCP
+utilized the functionality that ``normal`` staging areas can contain multiple
```

Comment:
Not sure if we need a clarification of some kind but I can't see `normal` staging areas mentioned anywhere else

Reply:
That should have been

```suggestion
utilized the functionality that ``complete`` staging areas can contain multiple
```